controllers/construct/build_statefulset_test.go

```diff
@@ -57,6 +60,24 @@ func TestMultipleCalls_DoNotCauseSideEffects(t *testing.T) {
 	})
 }
 
+func TestMongod_Container(t *testing.T) {
```

Comment:
unrelated test that ended up being written as part of investigation, no harm in keeping it I don't think!